### PR TITLE
Updated Crm Sample App

### PR DESCRIPTION
**Pull request by theneims**

### CODE REVIEW

**Comment by ShadowRonin** [comment]
What are you trying to accomplish with this PR?

File: src/App.js [hunk 1]
```diff
@@ -1,49 +1,67 @@
...
+} from '@amc-technology/davinci-api';
 
 class App extends Component {
   interactions = {};
 
   constructor() {
     super();
     this.initializeAMC();
-
+    // setAppHeight(100);
```
Comment (ShadowRonin) [inline src/App.js]: Dont commit commented code

File: src/App.js [hunk 2]
```diff
@@ -1,49 +1,67 @@
...
   }
 
   async initializeAMC() {
+    console.log('CRM awaiting bridgeScripts');
     await loadBridgeScripts([
-      'https://na82.salesforce.com/support/api/25.0/interaction.js',
+      'https://na132.salesforce.com/support/api/25.0/interaction.js',
```
Comment (ShadowRonin) [inline src/App.js]: Why was this changed? Does it need to be configurable?

Comment (theneims) [reply src/App.js]: This should be configurable based off the user's salesforce, "na132" is how I could access salesforce and it can be different from person to person I believe.
